packages/splits-sdk-react/src/hooks/liquid.ts

```diff
@@ -20,33 +19,30 @@ export const useCreateLiquidSplit = (): {
   error?: RequestError
 } => {
   const context = useContext(SplitsContext)
-  const splitsClient = getSplitsClient(context)
+  const splitsClient = getSplitsClient(context).liquidSplits
+
+  if (!splitsClient) throw new Error('Invalid chain id for liquid splits')
```

Comment:
are there any repercussions to throwing the error here? i guess it just means, if you have `useCreateLiquidSplit` on an invalid chain, we'll throw right away? instead of waiting until you try to call `createLiquidSplit`.

not sure on this. i mean on the surface it seems like a good change. just trying to think through if it makes life more difficult for devs.

anyway, think this is all good. just thinking out loud here.

Reply:
moved this inside the callback again